### PR TITLE
feat: Implement Mapbox for event location in F-Droid build variant

### LINKED ISSUE #1809: Implement Mapbox for adding event location in F-Droid build variant
**Describe the feature you'd like**
Implement Mapbox for adding event location in F-Droid build variant.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1809

GIF for the change:

Implement Mapbox for event location in F-Droid build variant

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -62,17 +63,19 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
 
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://api.eventyay.com/v1/"'
+            buildConfigField "String", "MAPBOX_ACCESS_TOKEN", "\"${MAPBOX_ACCESS_TOKEN}\""
```
Comment (iamareebjamal) [inline app/build.gradle]: Should be with Google Places API key, not in between base URLs

**Comment by iamareebjamal** [comment]
Note that app should not crash if there is no access token

File: app/build.gradle (hunk 1 above)
Comment (ShridharGoel) [reply app/build.gradle]: Updated.

**Comment by ShridharGoel** [comment]
> Note that app should not crash if there is no access token

Done.

**Pull request merged**